src/client/sandbox/node/element.js

```diff
@@ -98,6 +99,16 @@ export default class ElementSandbox extends SandboxBase {
             else if (el.hasAttribute(storedAttrName))
                 args[isNs ? 1 : 0] = storedAttrName;
         }
+        // NOTE: We simply remove the 'integrity' attribute because its value will not be relevant after the script
+        // content changes (http://www.w3.org/TR/SRI/). If this causes problems in the future, we will need to generate
+        // the correct SHA for the changed script.
+        // getAttributeCore returns stored 'integrity' attribute value. (GH-235)
```

Comment:
I think we can remove this line (only 105 line).  I don't see benefit of this comment.